ci/tests/disklayout/flexible_disk_layout_test.py

```diff
@@ -145,10 +145,11 @@ def fdl_0002_add_remove_partition_with_role_and_crosscheck_model_test():
         partitions_detected = False
         disk_guid = ''
         for path in unused_disks:
```

Comment:
1st: This test should be rewritten to allow to also use an unused disk on any other storagerouter in the cluster than the one on which the tests run
2nd: I don't understand why a path from an unused disk wouldn't be part of ```all_disks```, so this extra ```if``` statement makes no sense to me